WordPress/gutenberg
# Fix: The focus styles for tabPanel.

## What?
Fixes #60663 
When a tab panel is focused, the focus ring styles differ from those of the Tabs component. The main distinctions lie in the color and outline width, as no styles are added for the tab items, and user agent styles are applicable to them.

## Why?
This PR resolves the inconsistency between the focus styles for TabPanel and Tabs component.

## How?
This PR adds the box-shadow and outline styles for the class `.components-tab-panel__tab-content`

## Testing Instructions

1. Open the Storybook and navigate to the TabPanel component.
2. Either use the "Tab" key to navigate to the tabs or inspect the div below the Tabs labeled "Selected tab: Tab 1" and apply the focus:visible state to it.
3. You can observe the styles there and compare them with the Tabs components under experimental components as well.

### Testing Instructions for Keyboard
Nil

## Screenshots or screencast 

### Before
<img width="884" alt="image" src="https://github.com/WordPress/gutenberg/assets/55375170/452e9d4b-0548-49e4-b340-fa3050ee7910">


### After
<img width="995" alt="image" src="https://github.com/WordPress/gutenberg/assets/55375170/913b6b6b-7847-4b9b-8ccf-550837e68dec">
